testing/vcs/test_script_for_py.py

```diff
@@ -0,0 +1,15 @@
+import vcs
+
+list=["boxfill","meshfill","isofill","textcombined","texttable","vector"]
+for obj in list:
+    s=''
+    tc=''
+    if obj == 'textcombined':
```

Comment:
Let's extract the secondary graphics methods to a separate loop. Also drop the below logic.